sympy/printing/latex.py

```diff
@@ -2692,11 +2698,17 @@ def _print_MIMOFeedback(self, expr):
         inv_mat = self._print(MIMOSeries(expr.sys2, expr.sys1))
         sys1 = self._print(expr.sys1)
         _sign = "+" if expr.sign == -1 else "-"
-        return r"\left(I_{\tau} %s %s\right)^{-1} \cdot %s" % (_sign, inv_mat, sys1)
+        return r"\left(I_{\tau} %s %s\right)^{-1} \cdot %s" % (_sign, inv_mat,
+                                                               sys1)
 
     def _print_TransferFunctionMatrix(self, expr):
         mat = self._print(expr._expr_mat)
-        return r"%s_\tau" % mat
+        if expr.sampling_time == 0:
+            print_mat = r"%s_\tau" % mat
+        else:
+            print_mat = r"%s_k \text{, sampling time: } {%s}" % (mat,
+                                                           expr.sampling_time)
+        return print_mat
```

Comment:
I don't see any tests for the new printers. Where are testing the latex and pretty printing?